### DISCUSSION TITLE
Make retry cont configurable

### DISCUSSION THREAD

**Discussion by z3rone** [elementdavv/internet_archive_downloader#43] [Ideas]
I widness a lot of chunks that only download after 2nd try in the console log. It would be great if I could set a larger retry limit than 3 because this is reached quite often for me.

**Comment by dinkc64** [comment]
This the exact reason downloads fail for some, I wasted a derpload of time re-trying and re-trying...
Today I got curious and examined the code, content1.js,
    const TRYLIMIT = 3;         // each leaf retry limit

I changed this to:
    const TRYLIMIT = 13;         // each leaf retry limit

And now it never fails while downloading a book.  yay!
